src/js/index.js

```diff
@@ -50,11 +51,11 @@ function handleTags() {
 
 	if (tags.length === 0) {
 		history.pushState(null, '', window.location.pathname);
-		return;
```

Comment:
This return statement shouldn't have been dropped

Reply:
I've removed it since we do want the function to continue executing, since if we `return` here expired promos won't get filtered out in case the checkbox isn't selected.

Reply:
If that's the case, the `reduce` default value needs to be `tags.length===0` (as it was before the latest commit) - otherwise nothing will be displayed by default

Reply:
Done in 1ae91e384cf37d4a3f2c974e17b889b8c851d3bc.